CHANGELOG.rst

```diff
@@ -1,5 +1,67 @@
 .. _changelog:
 
+0.12.0
+------
+
+This is an important upgrade that enables v1.0 of ``renku-python`` (renku CLI)
+to work with the renku web interface, renkulab. The ``renku-python`` component
+in addition provides the service that manages project creation, datasets and
+knowledge graph metadata production. Therefore major changes were needed to all
+of the other components that make up the renku platform. For the most part, many
+of the changes will not be immediately visible to users of the renku web
+application.
+
+However, from this point on, new projects created on renkulab will use renku CLI
+>= v1.0. We therefore strongly urge all users to have a look at the myriad of
+excellent new features that this major release enables by checkout out the
+`release notes
+<https://github.com/SwissDataScienceCenter/renku-python/blob/1.0.0-release-notes/renku-release-notes-1.0.0.md>`_.
+Most notably, this release completely changes the storage and generation of the
+knowledge graph metadata, with vastly improved performance and functionality. We
+have gone to great lengths to ensure a smooth transition for older projects and
+for continuity in the CLI, but please report issues on
+https://renku.discourse.group or
+https://github.com/SwissDataScienceCenter/renku/issues.
+
+Detailed release notes follow below.
+
+Users:
+~~~~~~
+
+* Support for ``renku-python >= 1.0`` - plugins, workflows, speed 🥳 `read all
+  about it!
+  <https://github.com/SwissDataScienceCenter/renku-python/blob/1.0.0-release-notes/renku-release-notes-1.0.0.md>`_
+* Improved backwards compatibility for older projects - this means fewer
+  mandatory project updates and interruptions 🎯
+* Streamlined update dialog in project status view 🚀
+* Estimates of project update duration for peace-of-mind 😯
+* improved display of math formulas in markdown preview
+* more robust autosave infrastructure
+* support for project-template-defined parameter validation
+* support for icons and description for project templates
+* (bug fix) fix problems with dataset "add to project" button
+* partial support for custom CA certificates
+* experimental support for s3 storage in user sessions
```

Comment:
We decided to call this "cloud storage" and should be consistent with the wording.
```suggestion
* experimental support for cloud storage in user sessions
```